episodes/25-wrap-up.md

```diff
@@ -47,12 +47,18 @@ This exercise should take 5 minutes.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+::: instructor
+
+You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found on the website for the specific training.
```

Comment:
Does the email sent to trainers not include the survey links anymore @sheraaronhurt ?

Reply:
@brownsarahm I'm pretty sure it still does. I can edit the update to indicate it can be found in either the email or the website, but I'll wait and see if Sher has anything to say first.